website/docs/d/is_volumes.html.markdown

```diff
@@ -0,0 +1,154 @@
+		- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+		Nested scheme for **deleted**:
+			- `more_info` - (String) Link to documentation about deleted resources.
+			  - Constraints: The value must match regular expression `/^http(s)?:\/\/([^\/?#]*)([^?#]*)(\\?([^#]*))?(#(.*))?$/`.
+		- `device` - (Optional, List) Information about how the volume is exposed to the instance operating system.This property may be absent if the volume attachment's `status` is not `attached`.
+		Nested scheme for **device**:
+			- `id` - (Optional, String) A unique identifier for the device which is exposed to the instance operating system.
+		- `href` - (String) The URL for this volume attachment.
+		  - Constraints: The value must match regular expression `/^http(s)?:\/\/([^\/?#]*)([^?#]*)(\\?([^#]*))?(#(.*))?$/`.
+		- `id` - (String) The unique identifier for this volume attachment.
+		  - Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `/^[-0-9a-z_]+$/`.
+		- `instance` - (List) The attached instance.
+		Nested scheme for **instance**:
+			- `crn` - (String) The CRN for this virtual server instance.
+			- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+			Nested scheme for **deleted**:
+				- `more_info` - (String) Link to documentation about deleted resources.
+				  - Constraints: The value must match regular expression `/^http(s)?:\/\/([^\/?#]*)([^?#]*)(\\?([^#]*))?(#(.*))?$/`.
+			- `href` - (String) The URL for this virtual server instance.
+			  - Constraints: The value must match regular expression `/^http(s)?:\/\/([^\/?#]*)([^?#]*)(\\?([^#]*))?(#(.*))?$/`.
+			- `id` - (String) The unique identifier for this virtual server instance.
+			  - Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `/^[-0-9a-z_]+$/`.
+			- `name` - (String) The user-defined name for this virtual server instance (and default system hostname).
+			  - Constraints: The maximum length is `63` characters. The minimum length is `1` character. The value must match regular expression `/^-?([a-z]|[a-z][-a-z0-9]*[a-z0-9]|[0-9][-a-z0-9]*([a-z]|[-a-z][-a-z0-9]*[a-z0-9]))$/`.
+		- `name` - (String) The user-defined name for this volume attachment.
+		  - Constraints: The maximum length is `63` characters. The minimum length is `1` character. The value must match regular expression `/^-?([a-z]|[a-z][-a-z0-9]*[a-z0-9]|[0-9][-a-z0-9]*([a-z]|[-a-z][-a-z0-9]*[a-z0-9]))$/`.
+		- `type` - (String) The type of volume attachment.
+		  - Constraints: Allowable values are: `boot`, `data`.
+	- `zone` - (List) The zone this volume resides in.
+	Nested scheme for **zone**:
```

Comment:
zone means zone name. No need to create a list out of it

Reply:
done.